doc/uploads/process.adoc

```diff
@@ -0,0 +1,38 @@
+[#head]
+# Process from the upload to the fetch
+
+This documentation explains what happens after uploading the zip containing the details about the build.
+It is highly recommended to take a look to the <<HowItWorks.adoc#head, How It Works>> section before.
+
+## Uploading the zip
+
+### ExecutionResource.java
+-> First step is to upload the zip (https://your-ara-server.org/api/projects/cube-back/executions/upload).
+
+### ExecutionService.java
+-> The zip is then unzipped +
+-> All the (cycle and build) details are extracted +
```

Comment:
Maybe use the format : `All the {cycle + build}` (parenthesis are for misc details 😄 )

Same `are extracted` -> from where ?